Tribler/Core/Utilities/utilities.py

```diff
@@ -284,6 +286,33 @@ def parse_magnetlink(url):
     return (dn, xt, trs)
 
 
+def fix_torrent(file_path):
```

Comment:
For what I see, fix_torrent is used to fix torrents that are about to be loaded from disk (IE, user clicks on them)
Maybe fix_torrent could return the fixed bencoded data instead of modifying the on disk torrent. This way we don't silently alter user files and we reduce IO a _little_ bit.